src/main/java/uk/gov/hmcts/reform/sscs/services/ccd/UpdateCcdAppellantData.java

```diff
@@ -139,12 +139,12 @@ private boolean updateCcdAppellantName(Appellant gapsAppellant, Appellant existi
         boolean dataChanged = false;
         if (null != gapsAppellantName) {
             if (StringUtils.isNotBlank(gapsAppellantName.getFirstName())
-                && !gapsAppellantName.getFirstName().equals(existingCcdAppellantName.getFirstName())) {
+                && StringUtils.isBlank(existingCcdAppellantName.getFirstName())) {
```

Comment:
We don't want to update the Appellant/Rep/Appointee so simply updateCcdAppellantName method to be removed, similarly for reps and appointee

Reply:
Logic modified to ignore all values of name from Gaps